lib/auth/webauthncli/export.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webauthncli
+
+// HasPlatformSupport provides indirections for tests.
+var HasPlatformSupport = hasPlatformSupport
```

Comment:
I wouldn't do this here, push the var to lib/client - that's where we need it mocked. lib/auth/webauthncli should just expose a regular method, as before. This cuts the scope of the "mocking", so it's a bit more safe.

Reply:
https://github.com/gravitational/teleport/pull/16635/commits/a9ebb25fdec5f199878bca52698ff3bc1ae61960
i have done it from scratch, PTAL

Reply:
Thanks!